samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c

```diff
@@ -133,11 +133,9 @@ static void publish_telemetry() {
 
     const char *str = iotcl_create_serialized_string(msg, false);
     iotcl_telemetry_destroy(msg);
-    if (NULL != str) {

```

Comment:
Can't believe you need to remove:

if(NULL != str) {
}

is this collateral damage?

Reply:
Must have been a bugfix that was reverted, will reimplement